src/silverlabnwb/timings.py

```diff
@@ -0,0 +1,68 @@
+import abc
+
+import numpy as np
+import pandas as pd
+
+
+class LabViewTimings(metaclass=abc.ABCMeta):
+
+    def __init__(self):
+        self.pixel_time_offsets = None
+        self.cycle_time = None
+
+    @abc.abstractmethod
+    def _read_relative_times_file(self, file_path):
+        pass
+
+
+class LabViewTimingsPre2018(LabViewTimings):
+
+    def __init__(self, relative_times_path):
+        self._read_relative_times_file(relative_times_path)
+
+    def _read_relative_times_file(self, file_path):
+        raw_data = pd.read_csv(file_path, names=('RelativeTime', 'CycleTime'),
+                               sep='\t', dtype=np.float64) / 1e6
```

Comment:
Maybe a note here that we're converting to seconds for consistency? It's in the docstring of the `NWBFile`'s method but worth moving/adding here too, I think.